diagnostic.md

```diff
@@ -7,69 +7,81 @@ What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
 // your response here
+resources :books, only: [:index, :show]
+resources :books, except: [:new, :edit, :create, :delete, :update]
 ```
 
 In what file does the above code go?
 
 ```md
 // your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
 // your response here
+Does it make a working copy of the table you want to work in? I know it's a step that comes after db:create and before db:seed and db:example.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
 // your response here
+The first one creates a model? and the second creates a controller for things.
```

Comment:
The first command generates routes, a controller and model for foo. The second generates just a foo controller.